src/main/java/ca/wolfram/beta/symath/NegateNode.java

```diff
@@ -0,0 +1,38 @@
+package ca.wolfram.beta.symath;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import ca.wolfram.beta.symath.Expression.VMap;
+
+public class NegateNode extends OperationNode{
+
+	private NegateNode(List<MathNode> children) {
+		super(children);
+	}
+	
+    public static MathNode create(List<MathNode> nodes) {
+    	if (nodes.size() > 1) throw new RuntimeException("Negate Node only takes 1 node");
```

Comment:
I feel like we can have this check automatically through the operation constructor, since the enum type specifies the number of children